src/server/cmd/pachd/main.go

```diff
@@ -208,7 +208,7 @@ func doEnterpriseMode(config interface{}) (retErr error) {
 	defer func() {
 		if retErr != nil {
 			log.WithError(retErr).Print("failed to start server")
-			pprof.Lookup("goroutine").WriteTo(os.Stderr, 2)
+			pprof.Lookup("goroutine").WriteTo(os.Stderr, 2) //nolint:errcheck // already logging error above
```

Comment:
I guess the error the linter is returning is the error from WriteTo.  Not much we can do if you can't write to stderr though.